### DISCUSSION TITLE
Manifest for package with executable released in .tar.gz archive

### DISCUSSION THREAD

**Discussion by mloskot** [microsoft/winget-pkgs#101705] [Q&A] [answered]
I have submitted manifest for https://github.com/kubernetes-sigs/kustomize tool in PR #101695 ad the problem I've hit is that they release Windows executable binaries packaged in `.tar.gz`, e.g. https://github.com/kubernetes-sigs/kustomize/releases/download/kustomize%2Fv5.0.1/kustomize_v5.0.1_windows_amd64.tar.gz
This leads to the tarred executable being installed, i.e. `kustomize_v5.0.1_windows_amd64.tar.exe`

I have generated the manifest with `YamlCreate.ps1` choosing the portable as the installer type (an alternative could be zip), but there is no option for `.tar.gz `

Is there any way to tell winget to ungzip and untar before deploying the file?

I have searched through the existing manifests and there seem to be no such case of executable in `.tar.gz` archive.

**Comment by mdanish-kh** [comment, marked as answer]
At the moment, WinGet supports only `.zip` archive format.

- https://github.com/microsoft/winget-cli/issues/2899

**Reply by mloskot** [reply]
I see. It looks like the only way is to ask kustomize folks to release Windows binaries in zip.
Thanks for the reference to that issue.
